src/core/mod.rs

```diff
@@ -369,10 +376,31 @@ lazy_static! {
         StacksEpoch {
             epoch_id: StacksEpochId::Epoch21,
             start_height: 2000,
-            end_height: STACKS_EPOCH_MAX,
+            end_height: 3000,
```

Comment:
Any reason we don't use the constants for these? e.g. `BITCOIN_REGTEST_STACKS_21_BURN_HEIGHT`